feature-group-definitions/jpegxl.yml

```
@@ -0,0 +1,8 @@
+name: JPEG XL
+spec: https://jpeg.org/jpegxl/workplan.html
```

I'm looking into adding an entry for JPEG XL in browser-specs. That URL does not strike me as a good *spec* URL though, since it targets a page that lists several specs. The [file format spec](https://www.iso.org/standard/85253.html) seems a better entry point from a browser perspective: supporting the file format likely means you also support the underlying JPEG XL core coding system spec.

```suggestion
spec: https://www.iso.org/standard/85253.html
```